src/main/java/org/ozwillo/dcexporter/model/DcModelMapping.java

```diff
@@ -52,6 +52,7 @@
     private String cityField;
     private Object geoLocation;
 
+    private boolean isPrivate;
```

Comment:
The convention is to call such a variable `private` and not `isPrivate`. See https://docs.oracle.com/javase/tutorial/javabeans/writing/properties.html for instance.

Reply:
Property renamed to `privateDataSet` as public / private are reserved keywords